spec/models/user_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+
+describe User do
+  describe "validating user information" do
```

Comment:
in most cases `describe` part should describe which method are you testing and `context` part should describe conditions in which tests are run. For example:
```
describe User do
  describe '#valid?' do # instance methods starts with '#' and class methods starts with '.' (dot)
    context 'when email is missing' do
      #...
    end
  end
  
  describe '.find_mathing_letter' do
    context 'when there is a match'
    context 'when there is no match'
  end
end
```

Reply:
thank you, good advise, would try keep that in mind next time

Reply:
fixed